### PR TITLE
Update ballerina version to update 10

### PR DESCRIPTION
**Pull request by aashikam**
# Description

$title

### CODE REVIEW

**Review by niveathika — APPROVED** [review]

**Review by niveathika — DISMISSED** [review]

File: ballerina/tests/listener_test.bal [hunk 1]
```diff
@@ -14,84 +14,82 @@
...
+//     }
+// }
 
 // Using direct-token config for client configuration
 Client lisbaseClient = check new (sfConfigRefreshCodeFlow);
 string testRecordId = "";
 
 @test:Config {
-    enable: true
+    enable: false
```
Comment (NipunaRanasinghe) [inline ballerina/tests/listener_test.bal:92]: Any particular reason to disable the listeners tests on U10?

Comment (aashikam) [reply ballerina/tests/listener_test.bal:92]: Yeah Maheshika has not set the secrets correctly so the authentication fails

Comment (NipunaRanasinghe) [reply ballerina/tests/listener_test.bal:92]: Ack. In that case, better to create and issue for tracking purposes ftm. Please add the issue link in here once created.

**Review by NipunaRanasinghe — APPROVED** [review]

File: ballerina/tests/listener_test.bal (hunk 1 above)
Comment (aashikam) [reply ballerina/tests/listener_test.bal:92]: https://github.com/ballerina-platform/ballerina-library/issues/6908
https://github.com/ballerina-platform/ballerina-library/issues/6909

**Pull request merged**